hw/ip/dma/data/dma.hjson

```diff
@@ -50,24 +69,41 @@
       hwre:     "true", // enable `re` latched signal of software read pulse
       resval:   1,
       fields: [
-        { bits: "0", name: "READY", desc: "Transaction iss done"},
+        { bits: "0", name: "READY", desc: "Transaction is done"},
```

Comment:
you can use ENUM https://github.com/esl-epfl/x-heep/blob/main/hw/ip/i2s/data/i2s.hjson#L49